opentelemetry-exporters-newrelic/build.gradle.kts

```diff
@@ -3,13 +3,34 @@ dependencies {
     val openTelemetryVersion = "0.6.0"
 
     api("com.newrelic.telemetry:telemetry:$newRelicTelemetrySdkVersion")
+    implementation("org.slf4j:slf4j-simple:1.7.26")
```

Comment:
why move this up into a hard dependency? we should only use the API

Reply:
because I have no idea what I'm doing and I tried it and it didn't work but I bet IDEA was just caching something.  I'll fix it.